style.css

```diff
@@ -105,6 +105,44 @@ main {
   padding-bottom: 13rem;
 }
 
+/* burger-menue actions */
+.burger-menue {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  background-color: #3c3a39;
+  color: #fff5e1;
+  font-size: 32px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  line-height: 44px;
+}
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement because as I can see your design has some bugs related to toggling the mobile menu which is an **edge-case** but a potential issue which can cause *Bad User Experience*. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and video.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Video
[Screencast from 28-06-23 01:04:42 AM IST.webm](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/66476329/ac795446-c2c3-4ede-9d51-7a141764de40)

Reply:
> * [ ]  Please 🙏🏻  adhere to the project requirement because as I can see your design has some bugs related to toggling the mobile menu which is an **edge-case** but a potential issue which can cause _Bad User Experience_. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and video.
> 
> ### Requirement
> ![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)
> 
> ### Video
>  Screencast.from.28-06-23.01.04.42.AM.IST.webm

thanks for giving feedback but I did not understand which kind of bug it has @skyv26

Reply:
Hi @ZabihullahNooriWardak, It is a bug a minor one because it will not appear the person who directly open the website on desktop. But let's say any technical manager or a developer look into your portfolio and check your portfolio functionality then it makes a bad impression. You just have to hide this mobile menu when user go beyond 768px screen size, exactly like what you did for your hamburger icon.